cmd/metamorph.go

```diff
@@ -42,6 +42,10 @@ const (
 	DbModeDynamoDB = "dynamodb"
 )
 
+var (
+	unminedTxsPeriod = 2 * time.Minute
```

Comment:
This could be a constant like here: https://github.com/bitcoin-sv/arc/blob/f2dc16a3b1700d3ba179537957e269c72608132d/blocktx/client.go#L35

Reply:
done